dali/ft/filecopy.cpp

```diff
@@ -1054,7 +1056,7 @@ void FileSprayer::calculateMany2OnePartition()
         setCanAccessDirectly(curFilename);
         if (partSeparator)
         {
-            offset_t contentLength = cur.size - cur.xmlHeaderLength - cur.xmlFooterLength;
+            offset_t contentLength = (cur.size > cur.xmlHeaderLength + cur.xmlFooterLength ? cur.size - cur.xmlHeaderLength - cur.xmlFooterLength : 0);
```

Comment:
(cur.size > cur.xmlHeaderLength + cur.xmlFooterLength ? cur.size - cur.xmlHeaderLength - cur.xmlFooterLength : 0);

Can cur.size be > 0 , but < (xmlHeaderLength+xmlFooterLength) ?
e.g. when no header/footer.

If so, looks like this will incorrectly set contentLength to 0.